code/modules/research/techweb/all_nodes.dm

```diff
@@ -95,15 +95,15 @@
 	description = "When simple medicine doesn't cut it."
 	prereq_ids = list("imp_wt_surgery")
 	design_ids = list("surgery_lobotomy", "surgery_heal_brute_upgrade_femto","surgery_heal_burn_upgrade_femto","surgery_heal_combo","surgery_exp_dissection")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)

```

Comment:
Is this intentional? It isn't mentioned in the PR. Not that I have anything against it.

Reply:
i forgot that was in the same file, it was part of the big PR